### PR TITLE
Add Cypress tests

### PR DESCRIPTION
**Pull request by semarche-kaltura**

### CODE REVIEW

**Review by lianbenjamin — APPROVED** [review]

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```
Comment (MosheMaorKaltura) [inline cypress/support/commands.ts:1]: Lets remove this obsolete file

File: cypress/support/e2e.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```
Comment (MosheMaorKaltura) [inline cypress/support/e2e.ts:1]: also - is this relevant? if not lets remove it from git

File: src/components/info/index.tsx [hunk 3]
```diff
@@ -31,16 +31,23 @@ export class Info extends Component<MergedProps> {
...
-        </OverlayPortal>
+      <OverlayPortal>
+        <Overlay open onClose={onClick}>
+          <div className={[styles.infoRoot, styles[playerSize]].join(' ')} data-testid="infoRoot">
+            {broadcastedDate && (
+              <div className={styles.broadcastDate} data-testid="broadcastedDate">
+                {broadcastedDate}
+              </div>
+            )}
+            <div className={styles.entryName} data-testid="entryName">
```
Comment (MosheMaorKaltura) [inline src/components/info/index.tsx:42]: We should avoid changing the component code for testing

File: README.md [hunk 4]
```diff
@@ -28,12 +28,20 @@ yarn install
...
+Then, build the plugin
 
 ```javascript
 yarn run build
 ```
 
+### Testing
+
+The plugin uses `cypress` tool for e2e tests
+
```
Comment (MosheMaorKaltura) [inline README.md:40]: Write how to update ui conf if needed.

**Review by MosheMaorKaltura — APPROVED** [review]

**Pull request merged**